tasks/browsersync.js

```diff
@@ -3,9 +3,14 @@ import config from '../lib/config';
 import gulp from 'gulp';
 import browserSync from 'browser-sync';
 
-const domain = config.get('app.domain');
-const port = config.get('app.port') ? config.get('app.port') : null;
-const proxy = domain && port ? `${domain}:${port}` : domain;
+let proxy;
+if (process.env.BROWSERSYNC_PROXY) {
+  proxy = process.env.BROWSERSYNC_PROXY;
+} else {
+  const domain = config.get('app.domain');
```

Comment:
Shouldn't this be deprecated? It's not documented anywhere anymore, and I don't mind bumping major versions in favour of clean code + an updated `CHANGELOG.md` 👍

Reply:
Let me know if you disagree though.

Reply:
It is now deprecated 👍